### PR TITLE
remove references to cloud.gov

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

- noticed a "cloud.gov" reference in a PR review. Changes in this PR are found by doing a search in the codebase for "cloud.gov"

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,24 +23,6 @@ All new features should enter `main` through a pull request.
   the merge (if we were really committed to the gag this would use the word _may,_
   but it's not that funny).
 
-## Non-production deployments
```
Comment (benwarfield-usds) [inline CONTRIBUTING.md:30]: I'd say we should document what we actually are doing, but there was never much evidence that people read this, so...

Comment (timothybest-usds) [reply CONTRIBUTING.md:30]: we do have some documentation on the subject over at https://github.com/usds/prime-simplereport-docs

We also do not have evidence that any one reads the existing docs

**Review by benwarfield-usds — APPROVED** [review]
LGTM. The manifest files got nuked already, right?

**Comment by timothybest-usds** [comment]
> LGTM. The manifest files got nuked already, right?

yes it was removed in https://github.com/CDCgov/prime-simplereport/pull/198/files

**Pull request merged**